heading : [0, 20, 40, 60, 80, 100, 120, 140, 160, 180, 200, 220, 240, 260, 280, 300, 320, 340] 
u_d : [3, 5, 8, 10, 15, 20, 25, 30]
u_d_asv : [3, 5, 8, 10, 15, 20, 25, 30]
dcpa :  [-80, -50, -20, -10, 0, 10, 20, 50, 80]
d_detection : [50, 100]